test/functional/Java8andUp/src_110_up/org/openj9/test/annotation/ContainsRuntimeAnnotationTest.java

```diff
@@ -59,11 +60,11 @@
 
 	int notStableField = 0;
 
-	@Stable
 	@MyFieldAnnotation
 	@MyFieldAnnotation2
 	@MyFieldAnnotation3
 	@MyFieldAnnotation4
+	@Stable
```

Comment:
why is the position being changed?

Reply:
I figured it would loop through the annotations before it, if it was first it would find the annotation and exit immediately. I put an annotation with members after Stable and it didn't fail (before my changes).